core/lib/eth_client/src/clients/mock.rs

```diff
@@ -389,6 +389,8 @@ impl BoundEthInterface for MockEthereum {
         data: Vec<u8>,
         _contract_addr: H160,
         options: Options,
+        _max_fee_per_blob_gas: Option<U256>,
+        _blob_versioned_hashes: Option<Vec<H256>>,
```

Comment:
should `_max_fee_per_blob_gas` be a part of an options and `_blob_versioned_hashes` be part of the data?

Reply:
[`Options`](https://docs.rs/web3/latest/web3/contract/struct.Options.html) is the external struct from `web3`, we cannot add new fields there. But both fields are related and `_blob_versioned_hashes` is also the part of tx.

Reply:
Maybe we could define our own `Options` that would privately embed `web3::Options`? Currently the interface looks somewhat clunky, e.g. we provide two ways of grouping, which doesn't make much sense.